server/src/main/java/org/apache/druid/server/emitter/SwitchingEmitterConfig.java

```diff
@@ -0,0 +1,31 @@
+package org.apache.druid.server.emitter;
```

Comment:
Composing emitter's config is [here in the server module](https://github.com/apache/druid/tree/master/server/src/main/java/org/apache/druid/server/emitter)

But all the other emitters are [here in the core module](https://github.com/apache/druid/tree/master/core/src/main/java/org/apache/druid/java/util/emitter/core)

Reply:
it's probably fine with the current structure. I think, what needs to initialize the composite emitter is in `server` and the emitter logic itself is in `core`.